bindsnet/network/__init__.py

```diff
@@ -167,6 +167,28 @@ def save(self, file_name: str) -> None:
         """
         torch.save(self, open(file_name, 'wb'))
 
+    def load(file_name: str) -> None:
```

Comment:
This function doesn't do anything. You're using `torch.load()` and then discarding the result.

Reply:
You can remove this function and just use `bindsnet.network.load_network` instead.